client-app/pages/catalog.vue

```diff
@@ -87,14 +89,17 @@
         <div class="flex-grow">
           <div class="flex">
             <h2 class="text-gray-800 text-21 font-bold uppercase lg:my-px lg:text-25">
-              <span>{{ selectedCategory?.label }}</span>
+              <span v-html="$t('pages.search.header', [searchParams.keyword])" v-if="isSearchQuery"> </span>
```

Comment:
Use `<i18n-t />` instead of `v-html`
See examples in code or https://vue-i18n.intlify.dev/guide/advanced/component.html#slots-syntax-usage